views/partials/schedule.hbs

```diff
@@ -79,7 +79,6 @@
 
             <h6 class="event-schedule-speaker">Donato Borrello <span class="event-schedule-speaker-talk">The Nitty Gritty of Adaptive Video Transmuxing in JS</span></h6>
             <h6 class="event-schedule-speaker">Sarah Meyer <span class="event-schedule-speaker-talk">JavaScript Minus JavaScript</span></h6>
-            <h6 class="event-schedule-speaker">Matthew Beale <span class="event-schedule-speaker-talk">What you see is not what you get</span></h6>
```

Comment:
We originally put Matt here, but now this is an empty slot. Label this slot "Surprise!"